elastic/logs/challenges/many-shards-quantitative.json

```diff
@@ -86,6 +86,170 @@
+      "warmup-iterations": 10,
+      "iterations": 50,
+      "tags": ["esql", "limit0"]
+    },
+    {
+      "operation": "esql_basic_count_group_1_limit_0",
+      "clients": 1,
+      "warmup-iterations": 10,
+      "iterations": 50,
+      "tags": ["esql", "limit0"]
+    },
+    {
+      "operation": "esql_basic_count_group_2_limit_0",
+      "clients": 1,
+      "warmup-iterations": 10,
+      "iterations": 50,
+      "tags": ["esql", "limit0"]
+    },
+    {
+      "operation": "esql_basic_count_group_3_limit_0",
+      "clients": 1,
+      "warmup-iterations": 10,
+      "iterations": 50,
+      "tags": ["esql", "limit0"]
+    },
+    {
+      "operation": "esql_basic_count_group_4_limit_0",
+      "clients": 1,
+      "warmup-iterations": 10,
+      "iterations": 50,
+      "tags": ["esql", "limit0"]
+    },
+    {
+      "operation": "esql_time_range_and_date_histogram_two_groups_pre_filter_limit_0",
+      "clients": 1,
+      "warmup-iterations": 10,
+      "iterations": 50,
+      "tags": ["esql", "limit0"]
+    },
+    {
+      "operation": "esql_time_range_and_date_histogram_two_groups_post_filter_limit_0",
+      "clients": 1,
+      "warmup-iterations": 10,
+      "iterations": 50,
+      "tags": ["esql", "limit0"]
+    },
+    {
+      "operation": "esql_dissect_duration_and_stats_limit_0",
+      "clients": 1,
+      "warmup-iterations": 10,
+      "iterations": 50,
+      "tags": ["esql", "limit0"]
+    },
+    {
+      "operation": "esql_basic_count_group_0",
+      "clients": 1,
+      "warmup-iterations": 10,
+      "iterations": 50,
+      "tags": ["esql", "count"]
+    },
+    {
+      "operation": "esql_basic_count_group_1",
+      "clients": 1,
+      "warmup-iterations": 10,
+      "iterations": 50,
+      "tags": ["esql", "count"]
+    },
+    {
+      "operation": "esql_basic_count_group_2",
+      "clients": 1,
+      "warmup-iterations": 5,
+      "iterations": 20,
+      "tags": ["esql", "count"]
+    },
+    {
+      "operation": "esql_basic_count_group_3",
+      "clients": 1,
+      "warmup-iterations": 5,
+      "iterations": 10,
+      "tags": ["esql", "count"]
+    },
+    {
+      "operation": "esql_basic_count_group_4",
+      "clients": 1,
+      "warmup-iterations": 5,
+      "iterations": 10,
+      "tags": ["esql", "count"]
+    },
+    {
+      "operation": "esql_time_range_and_date_histogram_two_groups_pre_filter",
+      "clients": 1,
+      "warmup-iterations": 5,
+      "iterations": 20,
+      "tags": ["esql", "date_histogram"]
+    },
+    {
+      "operation": "esql_time_range_and_date_histogram_two_groups_post_filter",
+      "clients": 1,
+      "warmup-iterations": 5,
+      "iterations": 20,
+      "tags": ["esql", "date_histogram"]
+    },
+    {
+      "operation": "esql_dissect_duration_and_stats",
+      "clients": 1,
+      "warmup-iterations": 5,
+      "iterations": 20,
+      "tags": ["esql", "dissect"]
+    },
+    {
+      "operation": "esql_from_all_limit_50",
```

Comment:
I'm fine adding all these, but we can also start with just this query only.

Reply:
Should I remove the query cache stuff, too?

Reply:
The query cache stuff is only relevant to the non-ESQL queries. We added those for comparison to ESQL (ie. picked similar queries to compare how ESQL performs against _search). ESQL does not support a query cache, and the query cache is turned off for all tracks except elastic/logs, so we had to turn it off here too to make the comparison reasonable. If we do not include the _search queries in this benchmark, we do not need the query cache parts either.

Reply:
Personally I think comparing ESQL to _search is useful. On the other hand we do get a reasonable amount of that with the nyc_taxis track already.